Distance_3/include/CGAL/squared_distance_3_2.h

```diff
@@ -389,6 +390,27 @@ squared_distance(const Triangle_3<K> & t,
   return internal::squared_distance(pt, t, K());
 }
 
+template <class K>
+inline
+typename K::FT
+squared_distance(const Tetrahedron_3<K> & t,
+                 const Point_3<K> & pt) {
+  if (t.has_on_boundary(pt) || t.has_on_bounded_side(pt))
+      return K::FT(0);
+
+  const Triangle_3<K> t0 = {t[0], t[1], t[2]};
+  const Triangle_3<K> t1 = {t[0], t[1], t[3]};
+  const Triangle_3<K> t2 = {t[1], t[2], t[3]};
+  const Triangle_3<K> t3 = {t[0], t[2], t[3]};
+
+  const typename K::FT d0 = CGAL::squared_distance(t0, pt);
+  const typename K::FT d1 = CGAL::squared_distance(t1, pt);
+  const typename K::FT d2 = CGAL::squared_distance(t2, pt);
+  const typename K::FT d3 = CGAL::squared_distance(t3, pt);
```

Comment:
This may involve some redundant computation. I am not saying we shouldn't do this, it can be better to have the functionality and only try to optimize later, but we should be aware that this is probably not optimal.

Reply:
Could be. I dont really know how the computation between a point and a triangle is performed